creds.py

```diff
@@ -1,61 +1,68 @@
-BOT_TOKEN = 'Your bot token here'
+BOT_TOKEN = 'Insert bot token here'
```

Comment:
Just an FYI, in the latest version, this token is read using the `$SPARKIEEE_TOKEN` env variable. I'll merge that change over this one, so just make sure for future development you're using that.